pkg/proxy/proxy.go

```diff
@@ -116,7 +124,7 @@ func Deploy(sso *apiv1.SSO, oidcClient *api.Client, cookieSecret string) (*Proxy
 	secretVersion := computeSecretVersion(secret)
 	podTempl := v1.PodTemplateSpec{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      buildName(sso.GetName(), sso.GetNamespace()),
+			Name:      buildName(sso.GetName(), ""),
```

Comment:
Add the namespace looks not useful as the resource is only available in a given namespace.
Although adding it makes the name be more than 63 chars quite easily then we end up with a ugly truncated string.